caillef/dustzh.lua

```diff
@@ -1,14 +1,15 @@
 
 Config = {
     Map = "caillef.quakzh_dust2",
-    Items = { "caillef.wooden_crate", "caillef.barrels", "xavier.damage_indicator", "caillef.roboteye" }
+    Items = { "caillef.wooden_crate", "caillef.barrels", "xavier.damage_indicator", "caillef.roboteye", "k40s.gun_bullet" }
 }
 
 local DEBUG = false -- starts with a single player
 local SOUND = true
 local ROUND_DURATION = 240
 local SPAWN_BLOCK_COLOR = Color(136,0,252)
 local MAX_NB_KILLS_END_ROUND = 40
+local displayControls =  true -- display controls at start
```

Comment:
Maybe this should be named `displayInstructions` to better match with its usage.